remote/new.go

```diff
@@ -76,6 +76,13 @@ func NewImage(repoName string, keychain authn.Keychain, ops ...ImageOption) (*Im
 		ri.createdAt = imageOpts.createdAt
 	}
 
+	if imageOpts.mediaTypes != "" {
+		ri.requestedMediaTypes = imageOpts.mediaTypes
+	}
```

Comment:
```suggestion
	ri.requestedMediaTypes = imageOpts.mediaTypes
```
This accomplishes the same thing